test/turing_model.jl

```diff
@@ -0,0 +1,177 @@
+@testset "turing_model.jl" begin
+    kidiq = CSV.read(joinpath("..", "data", "kidiq.csv"), DataFrame)
+    wells = CSV.read(joinpath("..", "data", "wells.csv"), DataFrame)
+    roaches = CSV.read(joinpath("..", "data", "roaches.csv"), DataFrame)
```

Comment:
See my comment about artifacts. It also gives an example of a method where this "figure out path and load from CSV" logic can be moved into. Then, these lines would become
```
    kidiq = TuringGLM.dataset("kidiq.csv")
    [...]
```

Reply:
I will dive into the Artifacts stuff in new commits.